.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+Please add a meaningful title and a short description of your issue. It's hard to help you if you only add a trackback with no comments or a one-liner.
+
+Add the following details to your report. Especially if you are unsure about that to include. 
+
+- [ ] Your Home Assistant release, `hass --version`
```

Comment:
I think we should structure it as a report. No need for checkboxes. Instead of referring to "Your" we should refer as if the person is writing it. I would also suggest to include GitHub formatting stubs but remove the message they should use it.

**Home Assistant release (`hass --version`):**


**Component/platform:**


**Problem:**


**Expected:**


**Problem, relevant `configuration.yaml` entries and steps to reproduce**
```

```